target/linux/ramips/image/mt7620.mk

```diff
@@ -947,6 +964,25 @@ define Device/vonets_var11n-300
 endef
 TARGET_DEVICES += vonets_var11n-300
 
+define Device/tplink_re200-v1
```

Comment:
This should be before vonets_var11n-300, shouldn't it?

Reply:
Whops, yeah, of course!